src/db/schemas/DND5/DND5Spell.ts

```diff
@@ -57,6 +58,10 @@ export class DND5Spell extends Model<DND5Spell> {
     @Column({type: DataType.TEXT({length: 'medium'})})
     materials: string;
 
+    @AllowNull(true)
+    @Column
+    level: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
```

Comment:
tabellen machen KEINERLEI Validierung. Sie sind dumme Datenhalter
```suggestion
    level: number;
```